examples/basic_ops/basic_ops.jl

```diff
@@ -0,0 +1,54 @@
+using ITensors
+
```

Comment:
Would be a good idea to put a `let` ... `end` block around all the code to avoid global scope (which can cause some surprises to new Julia users)

Reply:
Would a `main()` function be ok?